### DISCUSSION TITLE
Hello, I would like to ask you about visualization

### DISCUSSION THREAD

**Discussion by Jinn02-25** [pyg-team/pytorch_geometric#3767] [Q&A]
Dear author I would like to ask you how to add visualization to dpgnn_classfication and what should be the input of tsne.I hope you can guide me in your busy schedule!

**Comment by rusty1s** [comment]
Can you clarify what you mean? We provide some examples regarding visualization of node embeddings [here](https://colab.research.google.com/drive/14OvFnAXggxB8vM4e8vSURUp1TaKnovzX?usp=sharing). Hopefully, this is useful to you :)
